src/quick_start/browser_extension.md

```diff
@@ -0,0 +1,104 @@
+# TLSNotary Browser Extension <a name="browser"></a>
+
+In this Quick Start we will prove ownership of a Twitter account with TLSNotary's browser extension.
+
+## Run Notary Server
+
+If the notary server isn't running already, run the notary server (in this repository: <https://github.com/tlsnotary/tlsn.git>):
+
+```shell
+cd notary-server
+cargo run --release
+```
+
+The notary server will now be running in the background waiting for connections.
+
+## Install Browser Extension (Chrome/Brave)
+
+1. Download the browser extension from
+<https://github.com/tlsnotary/tlsn-extension/releases/download/0.0.1/tlsn-extension-0.0.1.zip>
+2. Unzip  
+   ⚠️ This is a flat zip file, so be careful if you unzip from the command line, this zip file contains many file at the top level
+3. Open **Manage Extensions**: <chrome://extensions/>
+4. Enable `Developer mode`
+5. Click the **Load unpacked** button
+6. Select the unzipped folder
+
+![](images/extension_install.png)
+
+(Optional:) Pin the extension, so that it is easier to find in the next steps:
+
+![](images/extension_pin.png)
+
+## Websocket Proxy
+
+Since a web browser doesn't have the ability to make TCP connection, we need to use a websocket proxy server. You can either run one yourself, or use a TLSNotary hosted proxy.
+
+### Local Proxy
+
+To run your own websockify proxy locally, run:
+```
+git clone https://github.com/novnc/websockify && cd websockify
+./docker/build.sh
+docker run -it --rm -p 55688:80 novnc/websockify 80 api.twitter.com:443
+```
+Note the `api.twitter.com:443` argument on the last line.
+
+### Hosted proxy
+
+Or, you can simply use remote proxy at `ws://notary.efprivacyscaling.org:55688`
+
+1. Open the extension
+2. Click "Option"
+3. Update proxy URL and click "Save"
+
+<img width="478"  src="images/extension_proxy.png">
+
+## Self-signed Certificates
```

Comment:
Replace this section with the changes here: https://github.com/tlsnotary/progcrypto_workshop/pull/3/files

Reply:
I plan to do that in a second iteration.